examples/references.bib

```diff
@@ -24,3 +89,36 @@ @book{mcelreath2018statistical
 }
 
 
+@misc{rezende2016variational,
+      title={Variational Inference with Normalizing Flows}, 
+      author={Danilo Jimenez Rezende and Shakir Mohamed},
+      year={2016},
+      eprint={1505.05770},
+      archivePrefix={arXiv},
+      primaryClass={stat.ML}
+}
+
+
+@InProceedings{salimans2015mcmcandvi,
+  title={Markov Chain Monte Carlo and Variational Inference: Bridging the Gap},
+  author={Salimans, Tim and Kingma, Diederik and Welling, Max},
+  booktitle={Proceedings of the 32nd International Conference on Machine Learning},
+  pages={1218--1226},
+  year={2015},
+  editor={Bach, Francis and Blei, David},
+  volume={37},
+  series={Proceedings of Machine Learning Research},
+  address={Lille, France},
+  month={07--09 Jul},
+  publisher={PMLR},
+  pdf={http://proceedings.mlr.press/v37/salimans15.pdf},
+  url={https://proceedings.mlr.press/v37/salimans15.html},
+  abstract={Recent advances in stochastic gradient variational inference have made it possible to perform variational Bayesian inference with posterior approximations containing auxiliary random variables. This enables us to explore a new synthesis of variational inference and Monte Carlo methods where we incorporate one or more steps of MCMC into our variational approximation. By doing so we obtain a rich class of inference algorithms bridging the gap between variational methods and MCMC, and offering the best of both worlds: fast posterior approximation through the maximization of an explicit objective, with the option of trading off additional computation for additional accuracy. We describe the theoretical foundations that make this possible and show some promising first results.}
+}
+
+
+@misc{sklearn2020topicextraction,
```

Comment:
this entry is rendered a bit strangely right now, I think we should add an author like scikit-devs or something of the sort so the inline citation looks better. Or it might be also possible to use a custom text when citing, maybe with type `cite:label` I think it is (do check the docs please)

Reply:
does the rendered citation look better now?